.eslintrc-browser.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": "./.eslintrc-base.json",
+
+  "env": {
+    // Note: don't set `"browser": true`; code in "src/" should be compatible with
+    // non-browser environments like Node.js with a custom window implementation
+    // like jsdom. All browser globals should be taken from window.
```

Comment:
>
> In .eslintrc-browser.json
> <https://github.com/angular/angular.js/pull/14952#discussion_r72982735>:
>
> > @@ -0,0 +1,17 @@
> > +{
> > +  "extends": "./.eslintrc-base.json",
> > +
> > +  "env": {
> > +    // Note: don't set `"browser": true`; code in "src/" should be compatible with
> > +    // non-browser environments like Node.js with a custom window implementation
> > +    // like jsdom. All browser globals should be taken from window.
>
> I would be happier if this didn't affect tests tbh 😕
>

My thinking was that we might not want to make it harder to run unit tests
in non-browser environments. It's mostly about taking all browser globals
from window, not directly. Is that so bad?